fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/MessageValue.java

```diff
@@ -74,6 +75,17 @@ public static Object getFieldValue(@Nonnull MessageOrBuilder message, @Nonnull L
     @Nullable
     public static Object getFieldOnMessage(@Nonnull MessageOrBuilder message, @Nonnull String fieldName) {
         final Descriptors.FieldDescriptor field = findFieldDescriptorOnMessage(message, fieldName);
+        return getFieldOnMessage(message, field);
+    }
+
+    @Nullable
+    public static Object getFieldOnMessage(@Nonnull MessageOrBuilder message, int fieldNumber) {
+        final Descriptors.FieldDescriptor field = findFieldDescriptorOnMessage(message, fieldNumber);
+        return getFieldOnMessage(message, field);
+    }
```

Comment:
It would be really nice to add some documentation if possible (same for other newly-added methods).